main.go

```diff
@@ -8,6 +8,7 @@ import (
 	"errors"
 	"flag"
 	"fmt"
+	"github.com/metalmatze/signal/internalserver"
```

Comment:
tiny nit: This is an unnecessary diff

Reply:
Auto import 😅

Reply:
Fixed in [4c521e0](https://github.com/observatorium/api/pull/293/commits/4c521e0d51cbae0f67e1ed6343414bc359a142d3)